src/envo/Makefile

```diff
@@ -158,7 +158,7 @@ $(ONT)-base.owl: $(SRC) $(OTHER_SRC)
 # Full: The full artefacts with imports merged, reasoned
 $(ONT)-full.owl: $(SRC) $(OTHER_SRC)
 	$(ROBOT) merge --input $< \
-		reason --reasoner ELK --equivalent-classes-allowed all --exclude-tautologies structural \
+		reason --reasoner ELK --equivalent-classes-allowed none --exclude-tautologies structural \
```

Comment:
Do you want to use `--equivalent-classes-allowed asserted-only`?

Reply:
This is for you to decide! I mostly prefer to strictly forbid equivalents, but if it's ENVOs policy to allow asserted ones go ahead!! Thanks for looking!

Reply:
Thanks @matentzn ... the decision is not mine alone to make :)  
What do you think @cmungall @pbuttigieg ?